about/support.html.md

```diff
@@ -30,5 +30,16 @@ Customers on [Launch, Scale, and Enterprise plans](https://fly.io/plans) have ac
 
 Your support address is in the [Fly.io dashboard](https://fly.io/dashboard). Select your organization, and look for your Support email address in the Billing section.
 
+## Working with support
+We understand that describing software issues can sometimes be challenging. Providing a clear and detailed description of the problem, including any noticeable symptoms and relevant artifacts, can greatly assist us in identifying similar cases and expediting the resolution process. If we can quickly find a similar issue, we'll work to resolve it promptly. Conversely, if the problem is more complex, we'll involve specialized resources to address it.
```

Comment:
maybe "software and infrastructure issues" or "technical issues" since it's not always just software?